.travis.yml

```diff
@@ -16,6 +16,7 @@ script:
   # instruct bash to quit if there is a non zero response code (error)
   # https://github.com/commercetools/ui-kit/pull/284
   - set -e
+  - yarn run jest --projects jest.eslint.config.js --maxWorkers=4 --reporters jest-silent-reporter
```

Comment:
We also have stylelint

Reply:
fixed in [d022a222002535f2e3d8a0a062c527a9245c9436](https://github.com/commercetools/ui-kit/pull/434/commits/d022a222002535f2e3d8a0a062c527a9245c9436)